packaging/nntrainer.spec

```diff
@@ -392,16 +395,24 @@ popd
 %endif #unit_test
 
 %if 0%{?gcov:1}
-mkdir -p gcov-obj
-find . -name '*.gcno' -exec cp '{}' gcov-obj ';'
+builddir=$(basename $(pwd))
+gcov_obj_dir=$(pwd)/gcov-obj/$builddir
+mkdir -p $gcov_obj_dir
+pushd build
+# the files under Applications & nnstreamer & test directories are not line coverage target.
+find . ! -path "*/gcov-obj/*" ! -path "*/Applications/*" ! -path "*/nnstreamer/*" ! -path "*/test/*" -name '*.gcno' ! -name "meson-generated*" ! -name "sanitycheck*" -exec cp --parents '{}' "$gcov_obj_dir" ';'
+popd
+python3 gcov_trim_path.py $(pwd)/build $gcov_obj_dir
 %endif
 
 %install
 DESTDIR=%{buildroot} ninja -C build %{?_smp_mflags} install
 
 %if 0%{?gcov:1}
-mkdir -p %{buildroot}%{_datadir}/gcov/obj/%{name}
-install -m 0644 gcov-obj/* %{buildroot}%{_datadir}/gcov/obj/%{name}
+pushd gcov-obj
+find . -type d -exec install -d {} %{buildroot}%{_datadir}/gcov/obj/%{name}/'{}' ';'
+find . -type f -exec install -m 0644 {} %{buildroot}%{_datadir}/gcov/obj/%{name}/'{}' ';'
+popd
```

Comment:
These lines seem to be a duplicated works.
How about removing these lines and chaning L399 to `gcov_obj_dir=%{buildroot}%{_datadir}/gcov/obj/%{name}/"$builddir"`
Or is this path `gcov_obj_dir=$(pwd)/gcov-obj/$builddir` is necessary to trim the path of the files?

Reply:
No The gcov_obj_dir is not related with the file name prefix.
Actually the suggestion that you gave me is the Tizen platform team's guideline.
This pr is the old way but generate the gcov rpm file as Tizen platform teams requested.
I've been trying to apply your suggestion but in this way the gcov rpm is empty and I don't know why for now.
I will try apply your suggestion anyway. Thanks.